scene/resources/packed_scene.cpp

```diff
@@ -2306,7 +2306,7 @@ void PackedScene::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_get_bundled_scene"), &PackedScene::_get_bundled_scene);
 	ClassDB::bind_method(D_METHOD("get_state"), &PackedScene::get_state);
 
-	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "_bundled", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_INTERNAL), "_set_bundled_scene", "_get_bundled_scene");
+	ADD_PROPERTY(PropertyInfo(Variant::DICTIONARY, "_bundled", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_INTERNAL | PROPERTY_USAGE_NEVER_DUPLICATE), "_set_bundled_scene", "_get_bundled_scene");
```

Comment:
does this affect duplicating a Scene file from the FileSystem dock? I am under the assumption the editor has it's own process, but i may be wrong

Reply:
Not duplicating this resource could cause an indirect link between both PackedScenes, depending on how is used:

- if the variable is always overriden, ie `_bundle = temp_dictionary` then should be safe
- if members of `_bundle` are accessed and modified, ie `_bundle["connections"].insert(....)` then that operation would affect both Packed Scenes

I know this is not a typical operation, but at least i would like to confirm Godot Editor does not do it by itself under any situation

Reply:
The `_bundle` property is constructed on the fly, so effectively it can't be shared.